HISTORY.md

```diff
@@ -303,6 +303,22 @@ For Leveled Compaction users, `CompactRange()` with `bottommost_level_compaction
 * Updated xxHash source code, which should improve kXXH3 checksum speed, at least on ARM (#11098).
 * Improved CPU efficiency of DB reads, from block cache access improvements (#10975).
 
+* Java API extensions to improve consistency and completeness of APIs
```

Comment:
We have a new mechanism to track unreleased change history now. Which is to add a separate file in the corresponding sub directory in the `unreleased_history` directory. Instead of directly updating HISTORY.md. Can you move this to there.